codemod/base.py

```diff
@@ -26,6 +26,7 @@
 import sys
 import textwrap
 from math import ceil
+import tty, termios
```

Comment:
I've never used a mac, but if I'm not mistaken macOS X (which I guess we are talking about) is based on BSD Unix and therefore the terminal subsystem should be almost (if not completely) identical with Linux. Maybe you can try and see if just works there?
The rework adds support for Windows, I didn't test this either but it seemed trivial enough.